server/portal/utils/translations.py

```diff
@@ -92,3 +33,18 @@ def url_parse_inputs(job):
                 parsed_values.append(input)
             job['inputs'][key] = parsed_values
     return job
+
+
+def url_parse_input_v3(source_url):
```

Comment:
Remind me what this is for?

Reply:
I'll remove this. This was part of me refactoring the job submission portion.